### ISSUE TITLE
Stack overflow redirect

### ISSUE THREAD

**Issue by toyinolape** [googlecolab/colabtools#2326]
**Is your feature request related to a problem? Please describe.**
When you click on the search StackOverflow for errors that are specific to your dataset, It sometimes gives no result because the error contains detailed information regarding your dataset 

**Describe the solution you'd like**
The search phrase should eliminate the numbers after "(values)" or "(index)" in a bracket to help the search become more generic 

**Describe alternatives you've considered**
Adjusting the search phrase in the search bar and running again gives the expected results

**Additional context**
Length of values (44245) does not match length of index (146313) - gives no  results 
Length of values does not match length of index - gives adequate results

**Comment by craigcitro** [comment]
Agreed this would be nice, but there's no easy way for us to do this automatically; I'd recommend eliding parts of the error message once you're searching.
